RecoVertex/AdaptiveVertexFinder/python/inclusiveVertexing_cff.py

```diff
@@ -59,7 +59,12 @@
 inclusiveCandidateVertexingCvsL = cms.Sequence(inclusiveCandidateVertexingCvsLTask)
 
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
+pp_on_XeXe_2017.toModify(inclusiveVertexFinder, minHits = 10, minPt = 1.0)
+pp_on_XeXe_2017.toModify(inclusiveCandidateVertexFinder, minHits = 10, minPt = 1.0)
+pp_on_XeXe_2017.toModify(inclusiveCandidateVertexFinderCvsL, minHits = 10, minPt = 1.0)
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
-    e.toModify(inclusiveVertexFinder, minHits = 10, minPt = 1.0)
-    e.toModify(inclusiveCandidateVertexFinderCvsL, minHits = 10, minPt = 1.0)
+pp_on_AA_2018.toModify(inclusiveVertexFinder, minHits = 999, minPt = 999.0)
+pp_on_AA_2018.toModify(inclusiveCandidateVertexFinder, minHits = 999, minPt = 999.0)
+pp_on_AA_2018.toModify(inclusiveCandidateVertexFinderCvsL, minHits = 999, minPt = 999.0)
```

Comment:
is there any use of b-tagging in HI?

Reply:
@slava77 there is some use of b-tagging, yes, but only for a few primary datasets.   For previous heavy-ion datasets we have run the SVing on AOD.  There's certainly a case to made for leaving this in, although it is quite time-consuming.  On the other hand, there is a high probability that we end up modifying the track selection for b-tagging (the performance is quite sensitive to this), and end up rerunning it anyway.   As it stands currently, the track selection is way too loose, as we are currently still using pp selections (@abaty can you confirm that?).   This leads to a ton of fakes in central events, which may be responsible for the long processing time.  Same goes for the PF linking, btw.  I propose to have another look when the MVA selections are in place, which will hopefully be soon.    If the timing is not too unreasonable, we can re-enable inclusiveCandidateVertexFinder.

Reply:
OK. I hope the future updates will keep things still relatively close to pp